docs/tutorials/install_and_admin/geonode_install/setup_configure_httpd.txt

```diff
@@ -121,6 +120,11 @@ them:::
     chmod -Rf 777 /home/geonode/geonode/geonode/uploaded/layers
     sudo chown www-data:www-data /home/geonode/geonode/geonode/static_root/
 
+Dowload GeoNode data to be served by Apache. You will be prompted for confirmation::
```

Comment:
That text came from the original docs, I just moved it forward to prevent an error, but I can fix it anyway.